telegram_notifier.py

```
@@ -0,0 +1,125 @@
+import requests
+import os
+from dotenv import load_dotenv
+from app.models.payloads import *
+
+load_dotenv()  # Load environment variables from .env file
+
+class TelegramNotifier:
+    def __init__(self):
+        self.bot_token = os.getenv("TELEGRAM_BOT_TOKEN")
+        self.chat_id = os.getenv("TELEGRAM_CHAT_ID")
+
+        if not self.bot_token or not self.chat_id:
+            raise ValueError("Missing TELEGRAM_BOT_TOKEN or TELEGRAM_CHAT_ID in .env file")
```

Raising ValueError here is acceptable to fail fast if critical env vars are missing, but consider one of: (a) using a custom exception to make error handling clearer, (b) including the names of missing variables in the error message, or (c) deferring the check until send time if you prefer the notifier to be constructible in restricted contexts (e.g., tests).